pykusto/expressions.py

```diff
@@ -634,6 +631,10 @@ def assign_to(self, *columns: 'Column') -> 'AssignmentBase':
             return AssignmentFromColumnToColumn(columns[0], self)
         return ArrayExpression.assign_to(self, *columns)
 
+    # Used for mv-expand
+    def to_type(self, type_name: TypeName) -> 'ColumnToType':
```

Comment:
If it is only used in mv-expand, I think it is better as a separate function rather than as a method in Column object.

Reply:
Done